src/components/shapediver/ui/ParametersAndExportsAccordionComponent.module.css

```diff
@@ -32,3 +32,14 @@
 .acceptRejectBtn {
 	width: 70%;
 }
+
+:global(.mantine-Accordion-item) {
+	border-top: calc(0.0625rem*var(--mantine-scale)) solid var(--_item-border-color) !important;
+	border-radius: var(--_accordion-radius) !important;
+	margin-bottom: 0.5rem;
+}
+
+:global(.mantine-Accordion-control) {
+	border-radius: var(--_accordion-radius) !important;
+}
+
```

Comment:
@LemToUp I believe you did this because @mathieu-shapediver commented this: 

_Like we did on the platform, let’s avoid sticking the accordions together like here: 
...
Instead, let’s have a fully rounded corner border around each accordion and a space like we have between parameters:_

However, it's not a good solution to apply these global styles. What Mathieu requested can be implemented by slightly changing `ParametersAndExportsAccordionComponent.tsx` such that each `Accordion` only has one `Accordion.Item`. Please do that instead.
Accordion

Reply:
@LemToUp Here is the last change requested by alex, please let me know once you fix this point.

Reply:
@snabela I thought it's component scoped styles...
Refactored.